cli/popper/gha.py

```diff
@@ -157,7 +157,7 @@ def instantiate_runners(self):
                     a, self.workspace, self.env,
                     self.dry_run)
 
-    def run(self, action_name=None, reuse=False, parallel=False):
+    def run(self, action_name=None, with_dependencies=False):
```

Comment:
we can simplify the implementation of this feature by extending the [`Workfow.get_stages()`](https://github.com/systemslab/popper/blob/master/cli/popper/parser.py#L67) function so that it can be given an action where to start generating stages. In this way, the logic of running a workflow stays the same (i.e. no modifications would be needed).